### DISCUSSION TITLE
프로젝트 마무리 이후.. 기능과 유지보수에 대하여

### DISCUSSION THREAD

**Discussion by ryan-g00** [prgrms-fe-devcourse/NFE1_2_2_MUX#143] [D. 아이디어&레퍼런스]
추가로 구현해보고자 하는 기능과 유지보수가 필요한 부분이 있다면 정리해 봅시다!
아래 커멘트 달아주시면 제가 확인하고 본문에 하나씩 추가해볼게요~!

**Comment by hyunbin78** [comment]
전 TS변환하는 작업해보면서 공부하고 있는데 각자 해보시면 공부도 되고 도움 될 것 같아요!! 
그리고 플레이 리스트 부분도 하면 좋겠지만 안되겠죠..? ㅜㅜ

**Reply by ryan-g00** [reply]
저두 플레이 리스트 좋아용! zustand 써서 만들어보고 싶거든요 그것도 함 추가해보져!
